Resources/Prototypes/Nyanotrasen/Entities/Objects/Consumable/Food/candy.yml

```diff
@@ -22,6 +22,9 @@
   - type: Sprite
     sprite: Nyanotrasen/Objects/Consumable/Food/candy.rsi
     state: lollipop
+  - type: Clothing
+    slots: [mask]
+    sprite: Objects/Consumable/Smokeables/Cigarettes/cigarette.rsi
```

Comment:
this should probably not use the cigarette sprite
i feel the two items are distinct enough